src/main/config/config_streamer.c

```diff
@@ -50,6 +50,10 @@ uint8_t eepromData[EEPROM_SIZE];
 #  define FLASH_PAGE_SIZE                 ((uint32_t)0x4000)
 # elif defined(STM32F446xx)
 #  define FLASH_PAGE_SIZE                 ((uint32_t)0x4000)
+# elif defined(AT32F435ZMT7) || defined(AT32F435RMT7)
```

Comment:
We should move these to target.h

Reply:
See https://github.com/betaflight/betaflight/pull/12394